virttest/data_plane.py

```diff
@@ -0,0 +1,398 @@
+"""
+Autotest implementation of iothread manager classes.
+
+These classes represents different iothread allocation scheme.
+"""
+import heapq
+import re
+
+from avocado.utils import cpu
+from virttest.qemu_devices.qdevices import IOThread
+
+
+class MinQueue(object):
+    """Min heap."""
+
+    def __init__(self, item_list=None):
```

Comment:
Maybe  `item_list=[]`, unless Strings are also allowed?

Reply:
It is best not to use a mutable object as default argument and mutate it. If so, future calls will have the mutated object as default.